src/pretix/plugins/stripe/payment.py

```diff
@@ -643,6 +683,11 @@ def _handle_payment_intent(self, request, payment, intent=None):
                     amount=self._get_amount(payment),
                     currency=self.event.currency.lower(),
                     payment_method=request.session['payment_stripe_payment_method_id'],
+                    payment_method_options={
+                        'card': {
+                            'moto': self.is_moto(request, payment),
```

Comment:
Is it safe to send moto=False on a Stripe account that is not approved for MOTO?

Reply:
"Safe" as in "works" or "won't get you flagged for SAQ D"?

Reply:
Ideally both ;)

Reply:
Was thinking of "works", though